src/App.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+    <div id="app">
+        <SignIn v-if="this.$route.path === '/'" />
+        <div v-else-if="this.$route.params.currentWorkspace || this.$route.path === '/user'">
```

Comment:
This condition doesn't seem sustainable as pages that don't require the `currentWorkspace` parameter are added. I'll push up a commit better solution.

Reply:
Yeah long term I think you're right but for now I wouldn't let this stop me from merging this.